packages/cli/README.md

```diff
@@ -20,7 +20,7 @@ $ npm install -g @coveo/cli
 $ coveo COMMAND
 running command...
 $ coveo (-v|--version|version)
-@coveo/cli/1.5.0 linux-x64 node-v14.17.0
+@coveo/cli/1.5.0 darwin-x64 node-v14.15.4
```

Comment:
```suggestion
@coveo/cli/1.5.0 linux-x64 node-v14.17.0
```

Just to avoid unnecessary changes now, and later when the CI will go through again